src/core/blockchain/celo/stats.ts

```diff
@@ -1,8 +1,227 @@
-import { GenericStats, AccountStats } from '../types/stats';
+import {
+    GenericStats,
+    IStatValueType,
+    AccountStats,
+    IValidatorCard,
+    CardActionType
+} from '../types/stats';
 import { Client } from './client';
+import { Blockchain } from '../types/blockchain';
 
 export class Stats extends GenericStats<Client> {
     public getAccountDelegateStats(): AccountStats {
-        throw new Error('Not Implemented');
+        return {
+            topStats: topStatsValues,
+            chartStats: chartStatsValues,
+            secondaryStats: secondaryStatsValues
+        };
     }
 }
+
+// dummy data for stats
+
+export const secondaryStatsValues = [
+    {
+        title: 'Daily',
+        color: '#6A6A6A',
+        type: IStatValueType.AMOUNT,
+        data: {
+            value: '0.1200000',
+            tokenSymbol: 'cGLD',
+            blockchain: Blockchain.CELO
+        }
+    },
+    {
+        title: 'Monthly',
+        color: '#6A6A6A',
+        type: IStatValueType.AMOUNT,
+        data: {
+            value: '3.60000',
+            tokenSymbol: 'cGLD',
+            blockchain: Blockchain.CELO
+        }
+    },
+    {
+        title: 'Yearly',
+        color: '#6A6A6A',
+        type: IStatValueType.AMOUNT,
+        data: {
+            value: '43.200000',
+            tokenSymbol: 'cGLD',
+            blockchain: Blockchain.CELO
+        }
+    }
+];
+
+export const chartStatsValues = [
+    {
+        title: 'Available',
+        color: '#FFFFFF',
+        type: IStatValueType.AMOUNT,
+        data: {
+            value: '1000.00',
+            tokenSymbol: 'cGLD',
+            blockchain: Blockchain.CELO
+        }
+    },
+
+    {
+        title: 'Voting',
+        subtitle: 'locked',
+        color: '#00DAFF',
+        type: IStatValueType.AMOUNT,
+        data: {
+            value: '10000.00',
+            tokenSymbol: 'cGLD',
+            blockchain: Blockchain.CELO
+        }
+    },
+    {
+        title: 'Not-Voting',
+        subtitle: 'locked',
+        color: '#E91E63',
+        type: IStatValueType.AMOUNT,
+        data: {
+            value: '10000.00',
+            tokenSymbol: 'cGLD',
+            blockchain: Blockchain.CELO
+        }
+    },
+    {
+        title: 'Unlocking',
```

Comment:
Maybe use here translate of keys from translate

Reply:
Its dummy data for now, will adapt when the blockchain calls will be linked